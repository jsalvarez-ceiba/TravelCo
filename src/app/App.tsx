import * as React from 'react';
import { AppRouter } from 'app/AppRouter';
import { GlobalErrorBoundary } from './core/errors/GlobalErrorBoundary';



function App() {
  return (
    <GlobalErrorBoundary>
        <AppRouter />
    </GlobalErrorBoundary>
  );
}

export default App;
