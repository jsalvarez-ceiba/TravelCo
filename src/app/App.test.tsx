import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('App Test', () => {

    test('Render App ;)', () => {
        render(<App />);
    });

    /* test('Snapshot', () => {
        const page = render(<App/>);
        expect(page).toMatchSnapshot();
    }); */

})

