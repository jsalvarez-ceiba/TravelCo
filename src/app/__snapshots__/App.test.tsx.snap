// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="collapse"
        id="navbarToggleExternalContent"
      >
        <div
          class="bg-dark p-4"
        >
          <h5
            class="text-white h4"
          >
            Bienvenido a TravelCo
          </h5>
          <br />
          <ul>
            <a
              class="link"
              href="/"
            >
              <li
                class="li"
              >
                 
                Home
                 
              </li>
            </a>
            <a
              class="link"
              href="/places"
            >
              <li
                class="li"
              >
                 
                Destinos
                 
              </li>
            </a>
            <a
              class="link"
              href="/new"
            >
              <li
                class="li"
              >
                 
                Reservar
                 
              </li>
            </a>
            <a
              class="link"
              href="/search"
            >
              <li
                class="li"
              >
                 
                Buscar reserva
                 
              </li>
            </a>
          </ul>
        </div>
      </div>
      <nav
        class="navbar navbar-dark bg-dark"
      >
        <div
          class="container-fluid"
        >
          <button
            aria-controls="navbarToggleExternalContent"
            aria-expanded="false"
            aria-label="Toggle navigation"
            class="navbar-toggler"
            data-bs-target="#navbarToggleExternalContent"
            data-bs-toggle="collapse"
            type="button"
          >
            <span
              class="navbar-toggler-icon"
            />
          </button>
        </div>
      </nav>
      <div
        class="backgroundFlyHome"
      >
        <div
          class="flexCenter"
        >
          <div>
            <div
              class="card border-primary mb-3"
            >
              <div
                class="card-header text-center"
              >
                <h1>
                  TravelCo
                </h1>
              </div>
              <div
                class="card-body text-primary"
              >
                <h5
                  class="card-title text-center"
                >
                  Reserva tus vuelos
                </h5>
                <p
                  class="card-text text-center"
                >
                  Contamos con destinos nacionales e internacionales
                </p>
                <div
                  class="center"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          class="flexCenter"
        >
          <div
            data-aos="flip-left"
          >
            <div
              class="card text-dark bg-warning mb-3 "
            >
              <div
                class="card-header"
              >
                <h1
                  class="text-center"
                >
                  Experiencia
                </h1>
              </div>
              <div
                class="card-body"
              >
                <h5
                  class="card-title"
                >
                  Viajes inolvidables
                </h5>
                <p
                  class="card-text"
                >
                  Con nosotros tendras los viajes mas placenteros de tu vida
                </p>
              </div>
            </div>
          </div>
          <div
            data-aos="flip-left"
          >
            <div
              class="card text-white bg-primary mb-3"
            >
              <div
                class="card-header"
              >
                <h1
                  class="text-center"
                >
                  Tranquilidad
                </h1>
              </div>
              <div
                class="card-body"
              >
                <h5
                  class="card-title"
                >
                  Disfruta de la tranquilidad
                </h5>
                <p
                  class="card-text"
                >
                  En travelco garantizamos tu tranquilidad durante tus viajes
                </p>
              </div>
            </div>
          </div>
          <div
            data-aos="flip-left"
          >
            <div
              class="card text-white bg-danger mb-3"
            >
              <div
                class="card-header"
              >
                <h1
                  class="text-center"
                >
                  Seguridad
                </h1>
              </div>
              <div
                class="card-body"
              >
                <h5
                  class="card-title"
                >
                  Maxima seguridad para ti
                </h5>
                <p
                  class="card-text"
                >
                  Nos preocupamos por protegerte a ti y a tus seres queridos
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="collapse"
      id="navbarToggleExternalContent"
    >
      <div
        class="bg-dark p-4"
      >
        <h5
          class="text-white h4"
        >
          Bienvenido a TravelCo
        </h5>
        <br />
        <ul>
          <a
            class="link"
            href="/"
          >
            <li
              class="li"
            >
               
              Home
               
            </li>
          </a>
          <a
            class="link"
            href="/places"
          >
            <li
              class="li"
            >
               
              Destinos
               
            </li>
          </a>
          <a
            class="link"
            href="/new"
          >
            <li
              class="li"
            >
               
              Reservar
               
            </li>
          </a>
          <a
            class="link"
            href="/search"
          >
            <li
              class="li"
            >
               
              Buscar reserva
               
            </li>
          </a>
        </ul>
      </div>
    </div>
    <nav
      class="navbar navbar-dark bg-dark"
    >
      <div
        class="container-fluid"
      >
        <button
          aria-controls="navbarToggleExternalContent"
          aria-expanded="false"
          aria-label="Toggle navigation"
          class="navbar-toggler"
          data-bs-target="#navbarToggleExternalContent"
          data-bs-toggle="collapse"
          type="button"
        >
          <span
            class="navbar-toggler-icon"
          />
        </button>
      </div>
    </nav>
    <div
      class="backgroundFlyHome"
    >
      <div
        class="flexCenter"
      >
        <div>
          <div
            class="card border-primary mb-3"
          >
            <div
              class="card-header text-center"
            >
              <h1>
                TravelCo
              </h1>
            </div>
            <div
              class="card-body text-primary"
            >
              <h5
                class="card-title text-center"
              >
                Reserva tus vuelos
              </h5>
              <p
                class="card-text text-center"
              >
                Contamos con destinos nacionales e internacionales
              </p>
              <div
                class="center"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        class="flexCenter"
      >
        <div
          data-aos="flip-left"
        >
          <div
            class="card text-dark bg-warning mb-3 "
          >
            <div
              class="card-header"
            >
              <h1
                class="text-center"
              >
                Experiencia
              </h1>
            </div>
            <div
              class="card-body"
            >
              <h5
                class="card-title"
              >
                Viajes inolvidables
              </h5>
              <p
                class="card-text"
              >
                Con nosotros tendras los viajes mas placenteros de tu vida
              </p>
            </div>
          </div>
        </div>
        <div
          data-aos="flip-left"
        >
          <div
            class="card text-white bg-primary mb-3"
          >
            <div
              class="card-header"
            >
              <h1
                class="text-center"
              >
                Tranquilidad
              </h1>
            </div>
            <div
              class="card-body"
            >
              <h5
                class="card-title"
              >
                Disfruta de la tranquilidad
              </h5>
              <p
                class="card-text"
              >
                En travelco garantizamos tu tranquilidad durante tus viajes
              </p>
            </div>
          </div>
        </div>
        <div
          data-aos="flip-left"
        >
          <div
            class="card text-white bg-danger mb-3"
          >
            <div
              class="card-header"
            >
              <h1
                class="text-center"
              >
                Seguridad
              </h1>
            </div>
            <div
              class="card-body"
            >
              <h5
                class="card-title"
              >
                Maxima seguridad para ti
              </h5>
              <p
                class="card-text"
              >
                Nos preocupamos por protegerte a ti y a tus seres queridos
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
